resources/metadata/gitopsUpdateDeployment.yaml

```diff
@@ -179,3 +185,13 @@ spec:
           params:
             - name: tool
               value: kubectl
+    - image: k8s.gcr.io/kustomize/kustomize:v3.8.7
```

Comment:
I tried to come up with a proper integration test. Especially this part is not clear to my how to validate that the binary is available.